docs/index.md

```diff
@@ -55,6 +55,7 @@ Note that links to pull requests prior to public release of the code (4.0) direc
 - `4.5` Add support for Manage ([#61](https://github.com/ibm-mas/ansible-devops/pull/61))
 - `4.4` Add CP4D and DB2W playbooks ([#51](https://github.com/ibm-mas/ansible-devops/pull/51))
 - `4.3` Add support for playbook junit result generation ([#39](https://github.com/ibm-mas/ansible-devops/pull/39))
+- `4.3` Complete support for user workload monitoring ([#31](https://github.com/ibm-mas/ansible-devops/pull/31))
```

Comment:
This needs to be moved up to a 5.1 bulletpoint now  ... and the PR number is 100 not 31 now as well.

Reply:
Also, can you make an update in galaxy.yml to set the version to 5.1.0 as part of the PR too, because atm galaxy publication is still manual.  Until #13 is done we need to manually manage the version in there with each PR to master 👍